src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,374 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Customize media metadata (artist, album, title, artwork) and respond to media controls (play, pause, etc.) on the Web.
+
+{# wf_updated_on: 2017-02-06 #}
+{# wf_published_on: 2017-02-06 #}
+{# wf_tags: news,mediasession,play,pause #}
+{# wf_featured_image: /web/updates/images/2017/02/featured.png #}
+
```

Comment:
I **strongly** encourage also including a `{# wf_featured_snippet: blah blah blah #}`.

The featured snipped is used on the index pages as the primary description. It CAN include HTML and is your "hook" to get someone to read your article. The description at the top of the document is used as the meta description and is only used as the primary description if `wf_featured_snippet` isn't provided.